alt-api-ui/src/schedule-viewer/ScheduleBlock.tsx

```diff
@@ -17,17 +20,35 @@ type ScheduleBlockProps = {
   }
 }
 
+const settings = useReadLocalStorage<SettingsOptions>('settings')
```

Comment:
> ```
> Warning: Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons:
> 1. You might have mismatching versions of React and the renderer (such as React DOM)
> 2. You might be breaking the Rules of Hooks
> 3. You might have more than one copy of React in the same app
> See https://reactjs.org/link/invalid-hook-call for tips about how to debug and fix this problem. react.development.js:207:29
> ```

> ```
> Uncaught TypeError: dispatcher is null
>     useCallback React
>     useReadLocalStorage useReadLocalStorage.ts:11
>     <anonymous> ChoicePicker.tsx:15
> react.development.js:1644:2
> ```